src/python/pants/backend/python/util_rules/lockfile_diff.py

```diff
@@ -34,7 +29,7 @@
 
 @dataclass(frozen=True, order=True)
 class PythonRequirementVersion:
-    _parsed: LegacyVersion | Version
```

Comment:
I take it `LegacyVersion` is no longer supported as of packaging 22.0... is that possibly a breaking change for Pants users?

Reply:
Sounds good 👍  although given there's a chance of behaviour change maybe we could have a quick note in the release notes to give people a starting point for answering "why did this change?!"

Reply:
Done.